services/subscriber/service.go

```diff
@@ -20,6 +20,8 @@ const (
 	statWriteFailures = "writeFailures"
 )
 
+// PointsWriter is an interface for writing points to the database.
```

Comment:
"PointsWriter is an interface for writing points to a subscription destination."